sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -186,11 +264,11 @@ static int connect_device()
   // Connect to the Provisioning Service
   if ((rc = MQTTClient_connect(mqtt_client, &mqtt_connect_options)) != MQTTCLIENT_SUCCESS)
   {
-    printf("Failed to connect, return code %d\n", rc);
+    printf("Failed to connect, MQTTClient return code %d\n", rc);
     return rc;
   }
 
-  return 0;
+  return MQTTCLIENT_SUCCESS;
```

Comment:
I don't think our code should return Paho return codes. We could return something new (specific for samples) instead.

Reply:
Let's chat.  I don't like us returning a non-explained integer if an enum or defined macro is available.

Reply:
return rc;  ??